src/pages/api/dokumenter/[journalpostId]/[dokumentId]/forhandsvisning.ts

```diff
@@ -2,9 +2,16 @@ import { NextApiHandler } from "next";
 import { v4 as uuidv4 } from "uuid";
 import { withSentry } from "@sentry/nextjs";
 import { getSession } from "../../../../../lib/auth.utils";
+import { Stream } from "stream";
 
 const audience = `${process.env.SAF_SELVBETJENING_CLUSTER}:teamdokumenthandtering:${process.env.SAF_SELVBETJENING_SCOPE}`;
 
+export const config = {
```

Comment:
Denne trenger jeg nok ikke, tror jeg, dette er vel bare for request body slik jeg tolker det.